cli/src/commands/config/config.ts

```diff
@@ -36,9 +36,9 @@ class ConfigCommand extends Command {
       char: "u",
       description: "Relayer recipient",
     }),
-    lookupTable: Flags.string({
+    lookUpTable: Flags.string({
       char: "t",
-      description: "Look-up table.",
+      description: "LookUpTable.",
```

Comment:
```suggestion
      description: "Lookup table.",
```

ChatGPT confirms that "lookup table" is correct - not "look-up table". :stuck_out_tongue:  And definitely not "LookUpTable" (which is ofc good for variable names, but not for descriptions or documentation).

Reply:
done